platform_versioned_docs/version-24.1/secrets/overview.mdx

```diff
@@ -75,26 +75,26 @@ The ECS Agent uses the [Batch Execution role](https://docs.aws.amazon.com/batch/
 **IAM permissions**
 
 1. Add the [`AmazonECSTaskExecutionRolePolicy` managed policy](https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AmazonECSTaskExecutionRolePolicy.html).
-2. Add this inline policy:
+1. Add [this inline policy](https://github.com/seqeralabs/nf-tower-aws/blob/master/launch/secrets-policy-execution-role.json):
 
 ```json
     {
         "Version": "2012-10-17",
         "Statement": [
             {
                 "Sid": "AllowECSAgentToRetrieveSecrets",
-                "Action": [
-                    "secretsmanager:GetSecretValue"
-                ],
-                "Resource": [
-                    "arn:aws:secretsmanager:<YOUR_COMPUTE_REGION>:*:secret:*"
-                ],
-                "Effect": "Allow"
-            }
-        ]
+                "Effect": "Allow",
+                "Action": "secretsmanager:GetSecretValue",
+                "Resource": "arn:aws:secretsmanager:*:*:secret:tower-*"
```

Comment:
Why no longer defining the `<YOUR_COMPUTE_REGION>` to limit to a specific region?

Reply:
Thanks! Added region back in.